packages/components/src/components/icon/brand/ic-brand-ctrader.svg

```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M6 0a6 6 0 00-6 6v.174l29.914-4.721A5.977 5.977 0 0026 0H6zM30.532 2.068L25.044 32H26a6 6 0 006-6V6c0-1.504-.554-2.88-1.468-3.932z" fill="#B51E1E"/><path fill-rule="evenodd" clip-rule="evenodd" d="M25.044 32H6a6 6 0 01-6-6V6.174l29.914-4.721c.22.19.427.395.617.615L25.044 32z" fill="#E22526"/><path d="M9.007 17.071c0-.532.084-1.031.252-1.498a3.596 3.596 0 011.946-2.072c.466-.205.998-.308 1.596-.308.392 0 .751.037 1.078.112.327.065.644.163.952.294l-.434 1.666a5.27 5.27 0 00-.644-.196 3.366 3.366 0 00-.784-.084c-.616 0-1.078.191-1.386.574-.299.383-.448.887-.448 1.512 0 .663.14 1.176.42 1.54.29.364.789.546 1.498.546.252 0 .522-.023.812-.07.29-.047.555-.121.798-.224l.294 1.708a4.887 4.887 0 01-.91.266 6 6 0 01-1.204.112c-.672 0-1.25-.098-1.736-.294a3.411 3.411 0 01-1.204-.826 3.329 3.329 0 01-.686-1.218c-.14-.476-.21-.99-.21-1.54zM22.993 11.051v1.862h-2.926v7.84h-2.184v-7.84h-2.926v-1.862h8.036z" fill="#fff"/></svg>
```

Comment:
We have added this icon 4 times in this PR, can we just reuse it?

Reply:
each package uses its own `PlatformIcons` component and that is the reason we need to add the required icon in the packages that use it.